v0/destinations/serenytics/data/SerenyticsIdentifyConfig.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "destKey": "id",
+    "sourceKeys": "messageId",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "user_id",
+    "sourceKeys": "userId",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "context_ip",
+    "sourceKeys": ["context.ip", "request_ip"],
+    "required": false
+  },
+  {
+    "destKey": "first_name",
+    "sourceKeys": "firstName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "last_name",
+    "sourceKeys": "lastName",
+    "sourceFromGenericMap": true,
+    "required": false
+  },
+  {
+    "destKey": "age",
+    "sourceKeys": ["traits.age", "context.traits.age"],
+    "sourceFromGenericMap": true,
```

Comment:
As we are mentioning `sourceFromGenericMap` to true, do we need to pass the array in sourceKeys?

Reply:
updated